File: print.cpp
#include <iostream>
using namespace std;
void printno(int n){
  if (n){
    cout << n;
    printno(n-1);
  }
  else{

  }
 }

void jprintno(int i, int n){
  if ( i != n){
    cout << i;
    jprintno(i+1, n);
  }
}


int main(){
  int a;
  cin >> a;
  printno(a);
  cout << endl;
  return 0;
}

File: print.py
def printno(n):
    if n:
        print n
        printno(n-1)

def jprintno(i,n):
    if i != n:
        print i
        jprintno(i+1, n)

a = input()
printno(a)
jprintno(1,a+1)
